src/Device.ts

```diff
@@ -107,6 +107,10 @@ export function detectDevice(): BuiltinHandlerName | undefined
 		{
 			return 'Safari12';
 		}
+		// WeChat inline browser on iOS.
+		else if (browser.satisfies({ ios: { OS: '>=14.3', wechat: '>=0' } })) {
```

Comment:
Why not just ios OS: iOS WeChat >= 14.3? We should we need to deal with every app safari based browser?

Reply:
As I said, WeChat is the most popular IM App in China, nearly everyone uses this app for daily communication, it is one of the major browsers in fact just like safari/chrome(maybe more important than them). So in my opinion this individual rule for WeChat is a significant improvement. 
As for the rule itself, because WeChat just use WkWebView for its inline browser, and only iOS >=14.3 the WkWebView supports getusermedia API as well as other WebRTC APIs. In this case, the WeChat's version is not the main constraint, so I think this rule is just OK.

Reply:
Maybe I made a confusion statement, I mean iOS >=14.3, not WeChat >=14.3, its my fault :)